resources/services/waf/web_acls.go

```diff
@@ -250,6 +255,9 @@ func fetchWafWebACLLoggingConfiguration(ctx context.Context, meta schema.ClientM
 }
 func resolveWafWebACLLoggingConfigurationRedactedFields(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	conf := resource.Item.(*types.LoggingConfiguration)
+	if conf == nil {
```

Comment:
I don't think this is needed.

Reply:
Does marshalling a nil value give you valid JSON?